### ISSUE TITLE
Adapt text on Not found screen

### ISSUE THREAD

**Issue by acorbi** [OpenDevelopmentMekong/wp-odm_theme#1244]
Originally raised on https://github.com/OpenDevelopmentMekong/editorial_workstream/issues/15

**Comment by Huyeng** [comment]
Alex, updating the texts need to be translated.

**Comment by acorbi** [comment]
Very true, the texts are on https://www.transifex.com/open-development-mekong/odm-wp-odm_theme/translate ready to translate.

**Comment by acorbi** [comment]
@Huyeng

**Comment by Sotheavin07** [comment]
@Huyeng I think the sentence should  be _"The contents you are requesting do not seem to exist. Please try using the search function on the top-right area of the screen or browsing other country sites **on top-left area**. I liked to add this in order to inform user on where to look for other country sites.

**Comment by acorbi** [comment]
@Sotheavin07 I have adapted the text on https://www.transifex.com/open-development-mekong/odm-opendev-wptheme/translate/#km/enpo/134971308 as you request. Please coordinate the translation and let me know when it is done so I deploy it to PROD

**Comment by acorbi** [comment]
@Sotheavin07 I have adapted the text on https://www.transifex.com/open-development-mekong/odm-opendev-wptheme/translate/#km/enpo/134971308 as you request. Please coordinate the translation and let me know when it is done so I deploy it to PROD

**Comment by punwath** [comment]
Regarding to the string "The contents you are requesting do not seem to exist. Please try using the search function on the top-right area of the screen or browsing other country sites."  I think the word "or browsing other country sites" should be dropped because the user already has the purpose to find a particular data of a country. For example, a user may want to find ELCs dataset on Cambodia, and the user will not be able to find this data on other countries.

**Comment by Huyeng** [comment]
@punwath, yes, sounds a little confusing to the users when read the texts. How does everyone think?
